src/lib/qa.ts

```diff
@@ -4,10 +4,11 @@ import {
   validateSolanaAddress,
 } from '@hashflow/taker-js/dist/helpers/validation';
 import BigNumber from 'bignumber.js';
-import { computeLevelsQuote } from 'helpers/levels';
-import { convertFromDecimals, convertToDecimals } from 'helpers/token';
-import { Environment, PriceLevel, Token } from 'helpers/types';
-import { validateEnvironment, validateMakerName } from 'helpers/validation';
+
+import { computeLevelsQuote } from '../helpers/levels';
+import { convertFromDecimals, convertToDecimals } from '../helpers/token';
+import { Environment, PriceLevel, Token } from '../helpers/types';
+import { validateEnvironment, validateMakerName } from '../helpers/validation';
```

Comment:
I don't see any changes to .prettierrc ? are these eslint or prettier related changes?

Reply:
I added the `../` to the paths because these modules weren't found when running `tsc`. Running the script with the old paths worked, but I think when packaging as a module for calling as a library, there were some differences. Perhaps I did something wrong with my config, but I'm not sure.